run.py

```diff
@@ -182,6 +185,35 @@ def create_accent_phrases(text: str, speaker_id: int) -> List[AccentPhrase]:
             speaker_id=speaker_id,
         )
 
+    def decode_base64_waves(waves: List[str]) -> List[np.array]:
```

Comment:
値２つ返しているのでたぶん`Tuple[np.ndarray, int]`が正しそうです。
ただTupleをimportしないといけずちょっとややこしいのと、なくても動くので、消しちゃってマージします・・・！
（気づいたときに修正頂けるととても嬉しいです！）
```suggestion
    def decode_base64_waves(waves: List[str]):
```